dfdaemon/config/config_test.go

```diff
@@ -195,3 +196,14 @@ func (s *ConfigTestSuite) TestRegistry_initTLSConfig(c *check.C) {
 		}
 	}
 }
+
+func TestA(t *testing.T) {
```

Comment:
Maybe it's not a good idea to use local files on your own machine.

Reply:
Sorry, my mistake, i have commit some debug code.